src/common/Args.hpp

```diff
@@ -9,13 +9,37 @@
 namespace chatterino {
 
 /// Command line arguments passed to Chatterino.
+///
+/// All accepted arguments:
+///
+/// Crash recovery:
+///   --crash-recovery
+///   --cr-exception-code code
+///   --cr-exception-message message
+///   --cr-extra-memory=bytes
+///
+/// Native messaging:
+///   --parent-window
+///   --x-attach-split-to-window=window-id
+///
+/// -v, --verbose
+/// -V, --version
+/// -c, --channels=t:channel1;t:channel2;...
+///     --safe-mode
+///
+/// See documentation on `QGuiApplication` for documentation on Qt arguments like -platform.
 class Args
 {
 public:
     Args(const QApplication &app);
 
     bool printVersion{};
+
     bool crashRecovery{};
+    std::optional<uint32_t> exceptionCode{};
+    std::optional<QString> exceptionMessage{};
```

Comment:
Comment these two - exception code is platform-specific exception code from crashpad
message is text version of that code, potentially with extra context